sdk/core/src/http_client.rs

```diff
@@ -92,7 +94,8 @@ impl HttpClient for hyper::Client<HttpsConnector<hyper::client::HttpConnector>>
     }
 }
 
-#[async_trait]
+#[cfg(feature = "enable_reqwest")]
+#[async_trait(?Send)]
```

Comment:
This change appears to have a cascading effect. https://docs.rs/async-trait/0.1.48/async_trait/#non-threadsafe-futures

Reply:
cfg_attr to the rescue! I've disabled Send from wasm builds only. Thanks to:
https://github.com/dtolnay/async-trait/issues/159#issuecomment-819241829